### ISSUE TITLE
Linter doesn't honor .eslintrc file when project contains multiple folders

### ISSUE THREAD

**Issue by themaxx** [roadhump/SublimeLinter-eslint#63]
I've encountered a problem in which the linter ignores my .eslintrc file when my (SublimeText) project contains multiple folders. This appears to be due to the linter running within the first folder listed in the project, though I readily admit this isn't my area of expertise. If I set the first path entry to the root of the project—by simply re-ordering the list of folders in the sidebar to put the project root at the top—all of the lint errors in gulpfile.js go away because the .eslintrc file in the same directory is honored.

Don't know if this is an issue with this linter or the sublime linter plugin itself.

![image](https://cloud.githubusercontent.com/assets/1164081/8260530/f5d1226c-1689-11e5-9bcd-06952e6bd900.png)

**Comment by roadhump** [comment]
Can't repeat, can you try to reproduce it with some minimal files and post it here?

**Comment by yohanboniface** [comment]
Same here, I needed to explicitly add the path to the `.eslintrc` file folder and make it first in the project path list.
Here is the line ran in the console:

    SublimeLinter: eslint: crowdsourcing.js ['/usr/bin/eslint', '--format', 'compact', '--stdin', '--stdin-filename', '__RELATIVE_TO_FOLDER__']

Eslint vestion:

    $ eslint --version
    v0.23.0

Sublime-Text version: build 3083

Linter version:

    {"url": "http://sublimelinter.com", "dependencies": [], "description": "Interactive code linting framework for Sublime Text 3", "platforms": ["*"], "version": "3.5.0

Eslint plugin version:

    {"version": "1.5.4", "platforms": ["*"], "url": "https://github.com/roadhump/SublimeLinter-eslint", "dependencies": [], "sublime_text": ">=3000", "description": "This linter plugin for SublimeLinter provides an interface to ESLint"}

**Comment by roadhump** [comment]
Seems like ST bug :( . I've just posted https://github.com/SublimeTextIssues/Core/issues/911

Recommend to set 
```
{
    "linters": {
        "eslint": {
            "args": [
                "--stdin-filename",
                "@"
            ]
        }
    }
}
```
in your SublimeLinter Settings to prevent such issues.

**Comment by roadhump** [comment]
Fixed in v1.5.5
